support-workers/src/main/scala/com/gu/emailservices/FailedContributionEmailFields.scala

```diff
@@ -1,8 +1,20 @@
 package com.gu.emailservices
 
-import com.gu.salesforce.Salesforce.SfContactId
+object FailedEmailFields {
+
+  def contribution(email: String, identityUserId: IdentityUserId): EmailFields =
+    failedEmailFields("contribution-failed", email, identityUserId)
```

Comment:
I took email and identityUserId as parameters and passed directly on to failedEmailFields.  Although this could be simpler as a partial application, I don't think this code is complex enough to benefit from such an advanced technique.  Like this it's simpler and more compact than the old multi file class based approach.